reports/report_emailer.py

```diff
@@ -63,22 +43,27 @@ def average_delay_per_hour_graph(conn: psycopg2.extensions.connection, station_c
                 y=[i[1] for i in delays_per_hour_data])
     plt.xlabel("Hour of the Day")
     plt.ylabel("Average Delay Time")
+    plt.title("Average Delay Time for Delayed Trains")
 
     plt.savefig("average_delay_per_hour_graph.jpg")
 
 
-def get_station_name(conn: psycopg2.extensions.connection, station_crs: str):
+def get_station_name(conn: psycopg2.extensions.connection, station_crs: str) -> str:
```

Comment:
Nitpick : Could use str | None as the return type here, or alternatively (or both) mention that None is a possible return type in the docstring.

Reply:
Good catch Adam